test/integration/service/service_test.go

```diff
@@ -120,7 +120,7 @@ func Test_ExternalNameServiceDropsInternalTrafficPolicy(t *testing.T) {
 
 // Test_ConvertingToExternalNameServiceDropsInternalTrafficPolicy tests that converting a Service to Type=ExternalName
 // results in the internalTrafficPolicy field being dropped.This test exists due to historic reasons where the internalTrafficPolicy
-// field was being defaulted in older versions. New versions stop defauting the field and drop on read, but for compatibility reasons
+// field was being defaulted in older versions. New versions stop defaulting the field and drop on read, but for compatibility reasons
```

Comment:
(I would put these typo/error message text fixes into a separate commit, so that if it turns out that this PR breaks things and we have to revert it, we can revert the "Allow specifying ExternalTrafficPolicy for ClusterIP Services with ExternalIPs" commit, but still keep the "fix typos in comments" commit.)

Reply:
makes sense, moved typo fixes to a separate commit.